readme.md

```diff
@@ -39,11 +39,10 @@ fs.createReadStream('index.html').pipe(got.post('todomvc.com'));
 
 It's a `GET` request by default, but can be changed in `options`.
 
-#### got(url, [options], [callback])
+#### got([url][, options][, callback])
```

Comment:
I did not thought about it earlier but in fact, yes, it will be like `got({})`. Need check for "at least one argument"?